tests/functional/reset_password.js

```diff
@@ -151,16 +151,20 @@ define([
         .findById('fxa-reset-password-header')
         .end()
 
-        // ensure there is a back button
-        .findById('back')
+        // ensure there is a signin button
+        .findByCssSelector('a[href="/signin"]')
```

Comment:
You can always use the new helper [testElementExists](https://github.com/mozilla/fxa-content-server/blob/f1337b228ae027bdc988165f2b86b1487457e2da/tests/functional/lib/helpers.js#L982) here.